src/network/connection.h

```diff
@@ -555,15 +555,15 @@ class Peer {
 
 		bool isTimedOut(float timeout);
 
-		unsigned int m_increment_packets_remaining = 9;
-		unsigned int m_increment_bytes_remaining = 0;
+		unsigned int m_increment_packets_remaining = 0;
 
 		virtual u16 getNextSplitSequenceNumber(u8 channel) { return 0; };
 		virtual void setNextSplitSequenceNumber(u8 channel, u16 seqnum) {};
 		virtual SharedBuffer<u8> addSplitPacket(u8 channel, const BufferedPacket &toadd,
 				bool reliable)
 		{
-			fprintf(stderr,"Peer: addSplitPacket called, this is supposed to be never called!\n");
+			errorstream << "Peer::addSplitPacket called,"
+					<< " this is supposed to be never called!" << std::endl;
```

Comment:
The following sounds better, IMO:

` this is never supposed to be called!`

Reply:
If it's supposed to be never called, it's kinda pointless to change it.

Reply:
I commented before I realised that the PR has already been merged. Should've noted this before, really. Sorry. :)